### DISCUSSION TITLE
Is there a default sort column in prisma?

### DISCUSSION THREAD

**Discussion by jsoneaday** [prisma/prisma#9453] [Q&A] [answered]
I use BigInt table ids. But is there a default sort column or is it random?

**Comment by ryands17** [comment, marked as answer]
@jsoneaday 👋 

There's no default sort column in Prisma, it uses what the database does when you don't specify an `orderBy`. Mostly it's the `id`.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
